components/skeleton/skeleton-group-mixin.js

```diff
@@ -0,0 +1,19 @@
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
```

Comment:
Why does it need `RtlMixin`?

Reply:
it doesn't! just haven't cleaned that part up yet.